tensorbay/opendataset/CADC/catalog.json

```diff
@@ -0,0 +1,66 @@
+{
+    "BOX3D": {
+        "categories": [
+            { "name": "Bicycle" },
+            { "name": "Bus" },
+            { "name": "Car" },
+            { "name": "Pedestrian" },
+            { "name": "Pedestrian_With_Object" },
+            { "name": "Traffic_Guidance_Objects" },
+            { "name": "Truck" },
+            { "name": "Horse and Buggy" }
+        ],
+        "attributes": [
+            {
+                "name": "stationary",
+                "type": "boolean"
```

Comment:
Does CADC have docs to explain this attributes, if yes, add the explanation here as `description`